core/controlplane/config/templates/cloud-config-etcd

```diff
@@ -270,6 +270,7 @@ coreos:
             {{end -}}
 
             [Service]
+            Type=simple
```

Comment:
I appreciate it if you could revert this again, as we've done for the prev PR.

Reply:
Yes, no problem - I keep on adding it for my testing as my clusters fail to spin up properly without manual intervention. :)